css-grid-classes/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  font-family: sans-serif;
+}
+
+.container {
+  margin: auto;
+  width: 100vh;
+}
+
+.column-full {
+  padding-right: 47%;
```

Comment:
The only reason I made that declaration it was the only way that I could move the heading over since I had everything centered in the container. I will make the recommended changes.